balboa-agent/src/main/scala/com/socrata/balboa/agent/Config.scala

```diff
@@ -11,6 +11,8 @@ object Keys {
 
   lazy val SLEEP_MS = "balboa.agent.sleeptime"
 
+  lazy val INITIAL_DELAY = "balboa.agent.initialdelay"
+
```

Comment:
Yes.  it really provides no benefit because with the existing way Balboa Agent is instantiated and run it will always require this key.